bin/autobahn-router/src/server/http_server.rs

```diff
@@ -35,6 +35,7 @@ use router_lib::model::quote_response::{RoutePlan, SwapInfo};
 // make sure the transaction can be executed
 const MAX_ACCOUNTS_PER_TX: usize = 64;
 const MAX_TX_SIZE: usize = 1232;
+const DEFAULT_COMPUTE_UNIT_PRICE_MICROLAMPORTS: u64 = 10_000;
```

Comment:
should be 0 by default non ?

Reply:
this preserves the old behaviour. jupiter will estimate this if you don't pass anything in. a CU price of zero would cause a lot of transactions to go zero, not a good default behaviour